<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectingToServerDialog</class>
 <widget class="QDialog" name="ConnectingToServerDialog">
  <property name="windowTitle">
   <string>Connect to Server</string>
  </property>
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <layout class="QVBoxLayout" name="tableLayoutPanel1">
   <item>
    <widget class="QLabel" name="lblStatus">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Attempting to connect to ...</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar1">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
